Cargo.toml

```diff
@@ -13,3 +13,4 @@ license = "GPL-3.0-only"
 libloading = "0.6.1"
 log = "0.4.8"
 xenctrl-sys = { git = "https://github.com/Wenzel/xenctrl-sys.git" }
+xenvmevent-sys = { git = "https://github.com/Wenzel/xenvmevent-sys.git", optional = true }
```

Comment:
@arnabcs17b006 remove the `optional =  true` here, otherwise this crate will be skipped from the compilation process ;)